main/http_server.go

```diff
@@ -30,9 +30,9 @@ const (
 	JSONType     = "application/json"
 	HashLen      = 32
 
-	GatewayTimeout        = 60 * time.Second
-	ShutdownTimeout       = 30 * time.Second
-	BackendRequestTimeout = 30 * time.Second
+	GatewayTimeout        = 55 * time.Second
+	ShutdownTimeout       = 60 * time.Second
+	BackendRequestTimeout = 20 * time.Second
```

Comment:
Can you add a small comment how you arrived at this Timeouts?

Reply:
done